src/screens/Home/Home.tsx

```diff
@@ -126,15 +126,23 @@ function Home(props: HomeProps) {
         <div className={classes.section}>
           <Typography variant='h2'>
             {intl(words.tasks)}
-        </Typography>
-          <Link to={'/customers'}>
+          </Typography>
+          <Link
+            to={{
+              pathname: `/customers`,
+              state: { tabValue: CustomerStatus.METER }
+            }}>
             <HomeMenuItem
               label={intl(words.to_meter)}
```

Comment:
This is writing "To meter" and not "To Meter"?

Reply:
This has to do with the Adalo Metadata database. We can note it to the NPO to make sure that they capitalize it in the column